samples/csharp_dotnetcore/17.multilingual-bot/appsettings.json

```diff
@@ -1,5 +1,5 @@
 ﻿{
-  "botFilePath": "multilingual-bot.bot",
-  "botFileSecret": "",
+  "MicrosoftAppId": "",
+  "MicrosoftAppPassword": "",
   "translatorKey": "<Your translation key here>"
```

Comment:
Should we make translatorKey PascalCase like the other settings for consistency?

Reply:
fixed